app/cardrecord.js

```diff
@@ -77,6 +70,37 @@ class CardRecorder extends MyTrello {
     return actions;
   }
 
+  checkRecentMove(updates, currentTime) {
+    let moves = false;
+    let daysSinceUpdate = false;
```

Comment:
ah, yes, I'll change those both to null. That would behave the same in an conditional right?